tests/TestBase.js

```diff
@@ -11,4 +12,8 @@ module.exports = class TestBase {
   async cleanup(){
     await this.browser.close();
   }
+
+  async takeScreenShot(page){
+    await page.screenshot({ path: `reports/screenshots/example${this.count++}.png` });
```

Comment:
this will overwrite the images if you run the script multiple times. please use `UUID` instead